### ISSUE TITLE
acitool needs a README

### ISSUE THREAD

**Issue by brianredbeard** [appc/spec#459]
Was just about to reference `acitool` from my [minimal containers](https://github.com/brianredbeard/minimal_containers) repository but realized it had no `README` file.

**Comment by philips** [comment]
@brianredbeard Like you want a README.md here: https://github.com/appc/spec/tree/master/actool?

**Comment by brianredbeard** [comment]
Yup, or documentation somewhere for the tool that doesn't require downloading, compiling, and running just to get a sense of how to use it.
